src/lib.rs

```diff
@@ -1,18 +1,26 @@
-// pub mod slip;
-// pub mod transaction;
-// pub mod blockchain;
-// pub mod utxoset;
-// pub mod hashminer;
-// pub mod saito_router ;
-// pub mod mempool;
-// pub mod peer_pool;
-// pub mod peer;
-// pub mod keypair_store;
-// pub mod keypair;
-// pub mod crypto;
-// pub mod peer_policy;
-// pub mod concrete_peer_policy;
-// pub mod consensus_policy;
-// pub mod concrete_consensus_policy;
-// pub mod fork_policy;
-// pub mod concrete_fork_policy;
\ No newline at end of file
+/*!
```

Comment:
Whats' this doing here?

Reply:
This makes cargo doc pickup this comment and stick it on the main page of the docs. I actually snuck the docs onto production already if you wanna take a look: https://saito.io/docs/saito_rust/